### DISCUSSION TITLE
Github Graduation 2022 shows previously graduated

### DISCUSSION THREAD

**Discussion by PradyumnaKrishna** [community/community#17767] [GitHub Education]
My Graduation completes in 2022, but I mistakenly submitted a PR in last year and closed myself. Now when I create a PR this year it shows previously graduated and closes it. How to solve this issue?

My PR: https://github.com/education/GitHubGraduation-2022/pull/4106

**Comment by elisemoe** [comment]
Hi @PradyumnaKrishna 👋  please add a link to your PR in [this issue](https://github.com/education/GitHubGraduation-2022/issues/4290).
